pytest_copie/plugin.py

```diff
@@ -47,12 +61,19 @@ class Copie:
     counter: int = 0
     "A counter to keep track of the number of projects created."
 
-    def copy(self, extra_answers: dict = {}, template_dir: Optional[Path] = None) -> Result:
+    def git(self) -> plumbum.machines.LocalCommand:
```

Comment:
I think this should also get any args and kwargs from the "git" command and pass them to the git call as this is making all tests fail for a number of parameter issue.

Reply:
Right!  I tried to implement it this way however, in this case, the return type would not be anymore a [plumbum.macines.LocalCommand](https://plumbum.readthedocs.io/en/latest/api/machines.html#plumbum.machines.local.LocalCommand), but instead we would have to properly document what the [base run method](https://plumbum.readthedocs.io/en/latest/_modules/plumbum/commands/base.html#BaseCommand.run) needs to input, and what it outputs instead (different use-cases).  With the current implementation, we simply defer this documentation back to plumbum, therefore being a bit more lean IMO.

I can still do it in the other way if you prefer.  Let me know!